addons/l10n_pe_website_sale/views/templates.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <template id="partner_info" name="Peruvian partner">
+
+        <!-- show identification type -->
+        <div t-attf-class="mb-3 #{error.get('l10n_latam_identification_type_id') and 'o_has_error' or ''} col-xl-6">
+            <label class="col-form-label" for="l10n_latam_identification_type_id">Identification Type</label>
+            <t t-if="can_edit_vat">
+                <select
+                    name="l10n_latam_identification_type_id"
+                    t-attf-class="form-select #{error.get('l10n_latam_identification_type_id') and 'is-invalid' or ''}">
+                    <option value="">Identification Type...</option>
+                    <t t-foreach="identification_types or []" t-as="id_type">
+                        <option
+                            t-att-value="id_type.id"
+                            t-att-selected="id_type.id == int(identification) if identification else id_type.id == partner.l10n_latam_identification_type_id.id">
```

Comment:
why do we need a typecast?

Reply:
Sorry @jco-odoo 

I do not understand the question.

What do you see as incorrect here?